dojo/settings/settings.dist.py

```diff
@@ -375,6 +376,13 @@ def generate_url(scheme, double_slashes, user, password, host, port, path):
 AUTHORIZED_USERS_ALLOW_CHANGE = env('DD_AUTHORIZED_USERS_ALLOW_CHANGE')
 AUTHORIZED_USERS_ALLOW_DELETE = env('DD_AUTHORIZED_USERS_ALLOW_DELETE')
 
+# Setting SLA_NOTIFY_ACTIVE and SLA_NOTIFY_ACTIVE_VERIFIED to False will disable the feature
+SLA_NOTIFY_ACTIVE = False
+SLA_NOTIFY_ACTIVE_VERIFIED = True
```

Comment:
It would be great to add some comments to these settings keys.

Reply:
Yep.. they were there, and then not (i.e. they were in the actual PR desc).